src/styles/hc-2018/ternary.styl

```diff
@@ -292,6 +296,18 @@ input[type="submit"]:active{
     border: 1px solid var(--secondary-color);
 }
 
+#hack-cambridge-logo-object {
+    height: var(--two);
+}
+#hack-cambridge-wordmark, #hack-cambridge-wordmark * {
+    font-size: 21.3953315789pt;
```

Comment:
Out of interest, where are these numbers coming from, and can we express this more readably?

Reply:
Appears to be a really weird typo, as 24pt is correct…